lib/bunny/consumer_work_pool.rb

```diff
@@ -16,10 +16,14 @@ class ConsumerWorkPool
     attr_reader :threads
     attr_reader :size
 
-    def initialize(size = 1)
+    def initialize(size = 1, shutdown_timeout=nil)
```

Comment:
sorry to nitpick but please add a space between the argument and default value, like we do everywhere else